bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts

```diff
@@ -82,6 +82,79 @@ describe("GraphHitTestPolicy", () => {
     edge_stub.restore()
   })
 
+    describe("EdgesOnly", () => {
```

Comment:
Indent.

Reply:
I"d really like to have all the linter/codebase checks in the codebase job

Reply:
They is already `test_eslint` in codebase job and that job failed for the initial pusy (https://github.com/bokeh/bokeh/runs/1356605779?check_suite_focus=true) (along with bokehjs' jobs).